set.mm

```diff
@@ -501626,6 +501626,27 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       FUEBJNPQPRS $.
   $}
 
+  $( Integer ordering relation.  (Contributed by BTernaryTau, 24-Sep-2023.) $)
+  zltp1ne $p |- ( ( A e. ZZ /\ B e. ZZ ) ->
+        ( ( A + 1 ) < B <-> ( A < B /\ B =/= ( A + 1 ) ) ) ) $=
+    ( cz wcel wa c1 caddc co clt wbr cle wne cr zre peano2re ltlen sylan syl2an
+    wb zltp1le anbi1d bitr4d ) ACDZBCDZEZAFGHZBIJZUFBKJZBUFLZEZABIJZUIEUCAMDZBM
+    DZUGUJSZUDANBNULUFMDUMUNAOUFBPQRUEUKUHUIABTUAUB $.
+
+  $( Positive integer ordering relation.  (Contributed by BTernaryTau,
+     24-Sep-2023.) $)
+  nnltp1ne $p |- ( ( A e. NN /\ B e. NN ) ->
```

Comment:
Not a blocker (especially for a mathbox) but unless the version for `NN` or `NN0` is going to be widely used, I would tend to supply just the version for `ZZ` and then use `nn0zd` and similar theorems where it is applied.

Reply:
That's fair. IIRC I was trying to follow the example of similar groups of theorems like *ltp1le and *ltlem1 which exist for all 3.

Reply:
The closest thing I can give you to a consistent guideline is what I said before about what is widely used versus what is only used a few places. But I'm not completely sure how much we follow that.